### DISCUSSION TITLE
Also the download button in packs

### DISCUSSION THREAD

**Discussion by darylsimene** [Kyutefox/Iconify#17] [Ideas]
Is there a way that you could get the icons in pack? It would be really nice if we could see this as well. Your extension really help me! Thank you so much?!
![image](https://github.com/kyutefox/Iconify/assets/115441396/99be617d-affa-43ed-b313-5f091c75b514)

**Comment by MrJukeman** [comment]
@darylsimene We tried to achieve this result previously but where unsuccessful, we are still working on alternative for this feature ( specific to flaticon ), We will push update on this when completed.
